lib/human_player.rb

```diff
@@ -13,7 +13,7 @@ def get_opponent
     Mark::O if @mark == Mark::X 
   end
 
-  def get_next_move(board)
-    @display.ask_player_for_move(board)
+  def get_next_move
+    @display.ask_player_for_move(@mark)
```

Comment:
Yep, you can only do that if you have `attr_reader` or `attr_accessor` which essentially create getter/setter methods with that name.